### DISCUSSION TITLE
Suggest to request providing more details or clarify the context instead of fuzzy matching in 'lab chat'

### DISCUSSION THREAD

**Discussion by zhaohuijuan** [instructlab/instructlab#819] [Model Behavior]
In lab chat, when ask a question without enough detail or context, it answers the question with fuzzy matching, this behavior is 
 more like search engine than AI.  
Suggest lab chat to request providing more details or clarify the context instead of fuzzy match when answer such questions.

e.g. 
![Screenshot from 2024-03-14 14-45-48](https://github.com/instruct-lab/cli/assets/19279360/8786a1d8-61f3-49dc-8484-f9fa573fefec)
![Screenshot from 2024-03-14 14-47-07](https://github.com/instruct-lab/cli/assets/19279360/d5cb258d-db6b-4d1e-a607-72cce5b60606)

**Comment by hickeyma** [comment]
@zhaohuijuan I think `lab chat` is working as expected. Most models need more context when asked a question. Otherwise you will get unexpected output sometimes.

**Comment by zhaohuijuan** [comment]
@hickeyma Thanks for the clarification. 
Then maybe I can raise MR to provide more context/skill for this question.
